xml/obs_ag_installation_and_configuration.xml

```diff
@@ -270,18 +270,37 @@ systemctl start obswarden.service</screen>
       </para>
     </sect4>
     <sect4 xml:id="_aws_authentication_setup">
-      <title>AWS authentication credentials setup</title>
+      <title>AWS authentication – Server setup</title>
       <para>
-      For uploading images in AWS two tools are needed internally the <link xlink:href="https://aws.amazon.com/cli">aws CLI</link> and the <link xlink:href="https://github.com/SUSE/Enceladus/tree/master/ec2utils/ec2uploadimg">ec2uploadimg</link> (those are installed as dependencies).
-      For setting this up you have to configure both tools in the machine where the cloud upload workers will be running.
-    </para>
-    <para>
-      The configurations are stored in <emphasis>/etc/obs/cloudupload</emphasis> (the clouduploader home directory).
-      In this directory there is a <emphasis>.ec2utils.conf</emphasis> file which contains e.g. the helper instance AMIs (see this <link xlink:href="https://github.com/SUSE/Enceladus/blob/master/ec2utils/ec2utils.conf.example">example</link>).
-      In the <emphasis>/.aws/credentials</emphasis> file the AWS credentials of the OBS account are stored (see this <link xlink:href="https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html">documentation</link>).
-    </para>
-  </sect4>
-
+        Before you can start uploading images to the Amazon Web Services (AWS) you have to:
+        <orderedlist>
+          <listitem>
+            <para>Install the obs-cloud-uploader package</para>
+            <screen>
+              zypper in obs-cloud-uploader
+              zypper in aws-cli
+            </screen>
+          </listitem>
+          <listitem>
+            <para>Start the cloud upload services</para>
+            <screen>
+              rcobsclouduploadworker start
+              rcobsclouduploadserver start
```

Comment:
@mlschroe That's correct, right? Both services are needed?

Reply:
yes both are needed